code/datums/supplypacks/recreation_vr.dm

```diff
@@ -93,7 +93,7 @@
 	contains = list(
 			/obj/item/weapon/storage/smolebrickcase, /obj/item/weapon/storage/smolebrickcase,
 			)
-	cost = 200
+	cost = 49
```

Comment:
WHy an unround number? Almost every other thing in cargo is priced at multiple of 5

Reply:
I did it based on the price of a standard tub of legos.